src/tests/components/App.test.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import App from "../../components/App";
+
+describe("App", () => {
+  it("renders correctly", () => {
+    const { asFragment } = render(<App />);
+
+    expect(asFragment).toMatchSnapshot();
```

Comment:
Have a look at the snapshot for this file. Does it actually look like the component should once its rendered? Whats actually happening here is the snapshot isnt working as you need to call asFragment as it is a function. Just change this to `expect(asFragment()).toMatchSnapshot()`